docs/preparations.rst

```diff
@@ -35,35 +37,105 @@ lookups there to find your masterdata references. In the FMU metadata, we curren
   single applications shall own this definition.
 
 
-This is the content of ``_masterdata.yml`` from the Drogon example. Adjust to your needs:
+This is an example of ``_masterdata.yml`` from Drogon. Adjust to your needs:
 
 .. code-block:: yaml
 
+    # Note! Drogon is synthetic, so UUID's below are examples.
+
     smda:
       country:
-        - identifier: Norway
-          uuid: ad214d85-8a1d-19da-e053-c918a4889309
+        - identifier: Westeros
+          uuid: 00000000-0000-0000-0000-000000000000
       discovery:
         - short_identifier: DROGON
-          uuid: ad214d85-8a1d-19da-e053-c918a4889309
+          uuid: 00000000-0000-0000-0000-000000000000
       field:
         - identifier: DROGON
           uuid: 00000000-0000-0000-0000-000000000000
       coordinate_system:
-        identifier: ST_WGS84_UTM37N_P32637
-        uuid: ad214d85-dac7-19da-e053-c918a4889309
+        identifier: ST_WGS84_UTM37N_P12345
+        uuid: 00000000-0000-0000-0000-000000000000
       stratigraphic_column:
-        identifier: DROGON_HAS_NO_STRATCOLUMN
+        identifier: DROGON_2008
         uuid: 00000000-0000-0000-0000-000000000000
 
-Note that ``country``, ``discovery`` and ``field`` are lists. Most of us will only need one
-entry in the list, but in some cases, more will be required. E.g. if a model is covering
-more than one field, or more than one country.
+Note that ``country``, ``discovery`` and ``field`` are lists (can have more than one),
+while ``coordinate_system`` and ``stratigraphic_column`` are not.
+
+To find the information for your model, use `SMDA Viewer <https://opus.smda.equinor.com/smda_viewer/>`_. 
+You will see a number of topics, such as "Wellheader", "Well data" and below each topic
```

Comment:
Nit: it's "Well Header" in the site

Reply:
Removed the examples, they were redundant I think